pycbc/workflow/plotting.py

```diff
@@ -207,11 +208,19 @@ def make_veto_table(workflow, out_dir, vetodef_file=None, tags=None):
     if vetodef_file is None:
         vetodef_file = workflow.cp.get_opt_tags("workflow-segments",
                                            "segments-veto-definer-file", [])
+
```

Comment:
These lines should be within the if statement that pulls it from the workflow. It should be a File object if coming from elsewhere.